modules/c3-assisted-installer-completing-installation-nodes.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-completing-installation-nodes_{context}"]
+= Assigning node roles
+
+Delete the disconnected hosts and assign roles to the remaining hosts. Rename any hosts with names shorter than 63 characters.
+
+.Procedure
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters[{ai-full} web console], select the cluster and navigate to the *Host discovery* page. 
++
+If the Terraform scripts completed successfully, twelve hosts are listed on the page. Six hosts have the status *Disconnected* caused when you restarted the isntance on C3.
```

Comment:
What do you mean twelve hosts ? we can install cluster with 3 master ot even 1... ? not sure about this context.

Reply:
It was in the Oracle document.